src/app/app.css

```diff
@@ -484,7 +509,7 @@ input[type=number].datetime-picker__number-input {
 }
 
 .expandable-form__form-group.pf-c-form__group .pf-c-form__label-text {
-  color: #000000;
+  color: var(--pf-global--Color--100);
```

Comment:
We gonna need to have:

- Light: `var(--pf-global--Color--100)`
- Dark: `var(--pf-global--palette--white)` to make it the same color as the form group above.